7forecast.tex

```diff
@@ -398,10 +397,10 @@
 
 \begin{itemize}
 	\item 0 = Do nothing: this is the default and will invoke no special treatment for the forecast recruitments.
-	\item 1 = Multiplier on spawner-recruitment: the expected recruitment from the stock recruitment relationship is multiplied by this factor.
+	\item 1 = Multiplier on spawner-recruitment: the expected recruitment from the \gls{srr} is multiplied by this factor.
```

Comment:
should be "spawner--recruitment"

Reply:
@kellijohnson-NOAA  Are you saying that srr should be spawner recruitment relationship or that in the line there should be an en-dash in spawner-recruitment?